src/main/webapp/app/shared/table/GenomicIndicatorsTable.tsx

```diff
@@ -233,6 +233,7 @@ const GenomicIndicatorsTable = ({
                     );
                     await fetchGenomicIndicators(genomicIndicatorsPath);
                   }}
+                  noOptionsMessage={() => 'No such mutation in Mutations List'}
```

Comment:
```suggestion
                  noOptionsMessage={() => 'Please add this mutation in the Mutations List below first'}
```

@jfkonecn what do you think?

Reply:
Looks good to me